testcases/TC06.2-pass-1.md

```diff
@@ -41,7 +41,7 @@ result-descr: The image in the code sample is a decorative image from its <code>
 #
 # Then upload the code sample to the 'testfiles' folder
 # and provide the link (and only the url) below.
-sample: /testfiles/TF06/06.2-all-pass-1.html
+sample: /testfiles/TF06/06.2-pass-1.html
```

Comment:
I think this filename should be 06.2-1-pass-1.html. Reference the spreadsheet. The TC filename should be TC06.2-1-pass-1.html.